magento/pwa-studio
# Update Top Community Contributors

## Description

Updates the Top Community Contributors on the README for the 12.0.0 release.

## Related Issue







Closes PWA-2209

## Acceptance





Any core team member

### Verification Stakeholders



Any core team member

### Specification




## Verification Steps






1. View README.md
2. Verify numbers are accurate based on the `statistics.magento.engineering` link

#### Test scenario(s) for direct fix/feature





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
